pyedflib/edfwriter.py

```diff
@@ -136,29 +142,28 @@ def check_signal_header_correct(channels, i, file_type):
                                                                       str(ch['physical_max'])[:8]))
 
 
-
-def u(x):
+def u(x: bytes) -> str:
     return x.decode("utf_8", "strict")
 
 
-def du(x):
+def du(x: Union[str, bytes]) -> bytes:
     if isinstance(x, bytes):
         return x
     else:
         return x.encode("utf_8")
```

Comment:
I'm actually not sure these functions are necessary after deprecating Python 2.7.

Reply:
good idea! i created an issue: https://github.com/holgern/pyedflib/issues/265